src/workspace.js

```diff
@@ -1548,22 +1548,27 @@ module.exports = class Workspace extends Model {
   Section: Pane Locations
   */
 
+  // Essential: Get the {WorkspaceCenter} at the center of the editor window.
   getCenter () {
     return this.paneContainers.center
   }
 
+  // Essential: Get the {Dock} to the left of the editor window.
   getLeftDock () {
     return this.paneContainers.left
   }
 
+  // Essential: Get the {Dock} to the right of the editor window.
   getRightDock () {
     return this.paneContainers.right
   }
 
+  // Essential: Get the {Dock} below the editor window.
   getBottomDock () {
     return this.paneContainers.bottom
   }
 
+  // Essential: Get an {Array} of the current {WorkspaceCenter} and {Dock}s.
```

Comment:
Are we sure we want to make this method part of the public API?

I personally find this method's behavior a bit surprising. Specifically, I find it odd that this method returns a heterogenous collection of objects in which the objects have no explicit shared interface. I first ran into this in https://github.com/atom/atom/pull/14460#issue-229489706:

> It seems like there's an interface that we expect both `Dock` and `WorkspaceCenter` to implement, but I don't see that interface explicitly described or enforced anywhere. I _think_ that lack of explicit interface is a cause of the bug seen in #14442. There may be other bugs that are lurking due to code that implicitly expects a `WorkspaceCenter` and a `Dock` to provide a common interface.

Without an explicit interface that is shared by `WorkspaceCenter` and `Dock`, anyone using `Workspace::getPaneContainers()` will likely have to first check the type of each object in the collection before invoking any methods on that object.

If that sounds as odd to you as it does to me, I recommend excluding this method from the public API for now.